src/features/surveys/components/SurveyEditor/blocks/OpenQuestionBlock.tsx

```diff
@@ -1,13 +1,235 @@
+              fullWidth
+              InputProps={{
+                inputRef: titleRef,
+                sx: { fontSize: theme.typography.h4.fontSize },
+              }}
+              label={intl.formatMessage({
+                id: 'misc.surveys.blocks.common.empty',
+              })}
+              margin="normal"
+              onChange={(ev) => setTitle(ev.target.value)}
+              onKeyDown={(evt) => handleKeyDown(evt)}
+            />
+            <TextField
+              defaultValue={element.description}
+              fullWidth
+              InputProps={{ inputRef: descriptionRef }}
+              label={intl.formatMessage({
+                id: 'misc.surveys.blocks.common.description',
+              })}
+              margin="normal"
+              onChange={(ev) => setDescription(ev.target.value)}
+              onKeyDown={(evt) => handleKeyDown(evt)}
+            />
+            <TextField
+              defaultValue={
+                element.response_config.multiline === true
+                  ? FIELDTYPE.MULTILINE
+                  : FIELDTYPE.SINGLELINE
+              }
+              fullWidth
+              InputProps={{ inputRef: typeConfigRef }}
+              label={intl.formatMessage({
+                id: 'misc.surveys.blocks.open.textFieldType',
+              })}
+              margin="normal"
+              onChange={(event) => {
+                handleSelect(event),
+                  setResponseConfig(
+                    event.target.value === FIELDTYPE.MULTILINE ? true : false
+                  );
+              }}
+              select
+              SelectProps={{
+                MenuProps: { disablePortal: true },
+              }}
+              sx={{ alignItems: 'center', display: 'flex' }}
+              value={typeField}
+            >
+              {Object.values(FIELDTYPE).map((value) => (
+                <MenuItem key={value} value={value}>
+                  {value === 'singleLine' ? (
+                    <AbcIcon sx={{ marginRight: '10px' }} />
+                  ) : (
+                    <SortIcon sx={{ marginRight: '10px' }} />
+                  )}
+                  <Msg id={`misc.surveys.blocks.open.${value}`} />
+                </MenuItem>
+              ))}
+            </TextField>
+          </Box>
+        </ClickAwayListener>
+      )}
+      {!inEditMode && (
+        <ClickAwayListener
+          onClickAway={() => {
+            const element = createElementToUpdate();
+            onEditModeExit(element);
+            setFocus(null);
+          }}
+        >
+          <Box onClick={() => onEditModeEnter()}>
+            <Typography
+              component="h4"
+              marginBottom={2}
+              onClick={() => setFocus('title')}
+              variant="h4"
+            >
+              {element.question ? (
+                element.question
+              ) : (
+                <Msg id="misc.surveys.blocks.common.empty" />
```

Comment:
My suggestion in my last review was to use an 'empty'-message that is specific to to each block type, so for open question it would be "Untitled open question". Do you think that is a good solution or do you want to keep "Untitled block" like it is here?